src/app/content/Content.tsx

```diff
@@ -1,24 +1,40 @@
-import React from 'react';
-import { useGetCourses } from '../../fetchers';
+import { Box, Skeleton } from '@chakra-ui/react';
+import { Term, useGetCourse } from '../../fetchers';
+import { CourseInfo } from './components/Course';
+import { SectionsContainer } from './containers/Section';
 
 export interface ContentProps {
   /**
    * Content
    * Subject to change
    */
   content?: string;
+  pid?: string;
+  term?: Term;
 }
 
 /**
  * Primary UI component for content
  */
-export const Content: React.FC<ContentProps> = ({
-  content
-}) => {
-  const { data } = useGetCourses({ term: '202101' });
+export function Content({ pid, term = '202101' }: ContentProps): JSX.Element {
```

Comment:
I thought setting the term was supposed to pass through props?

Reply:
It is, it just has a default value set to `202101`. My guess is this can be removed once the header is implemented and a term will always be selected via the buttons.

Reply:
ah i see kk

Reply:
Yeah. I'll add a note to switch it over once we can pass that in.